test/test_torch.py

```diff
@@ -5832,35 +5846,35 @@ def test_pow(self, device):
                     res1 = torch.pow(m1[4], num)
                     res2 = res1.clone().zero_()
                     for i in range(res2.size(0)):
-                        res2[i] = math.pow(m1[4][i], num)
-                    self.assertEqual(res1, res2)
+                        res2[i] = pow(m1[4][i], num)
```

Comment:
What's going on here?

Reply:
math.pow doesn't work for complex and cmath only had exp, so using python  `pow` here